packages/analytics-browser/src/config.ts

```diff
@@ -168,24 +168,18 @@ export class BrowserConfig extends Config implements IBrowserConfig {
 export const useBrowserConfig = async (apiKey: string, options?: BrowserOptions): Promise<IBrowserConfig> => {
   const defaultConfig = getDefaultConfig();
 
-  // create cookie storage
-  const domain = options?.disableCookies ? '' : options?.domain ?? (await getTopLevelDomain());
-  const cookieStorage = await createCookieStorage<UserSession>({ ...options, domain });
-  const previousCookies = await cookieStorage.get(getCookieName(apiKey));
-  const queryParams = getQueryParams();
-
   // reconcile user session
-  const deviceId = options?.deviceId ?? queryParams.deviceId ?? previousCookies?.deviceId ?? UUID();
-  const lastEventTime = options?.lastEventTime ?? previousCookies?.lastEventTime;
-  const optOut = options?.optOut ?? Boolean(previousCookies?.optOut);
-  const sessionId = options?.sessionId ?? previousCookies?.sessionId;
-  const userId = options?.userId ?? previousCookies?.userId;
+  const deviceId = options?.deviceId ?? UUID();
+  const lastEventTime = options?.lastEventTime;
+  const optOut = options?.optOut;
+  const sessionId = options?.sessionId;
+  const userId = options?.userId;
 
   return new BrowserConfig(apiKey, {
     ...options,
-    cookieStorage,
+    cookieStorage: options?.cookieStorage,
```

Comment:
Why make cookieStorage and domain different from others?

Reply:
Ah yeah good point. I can make it consistent

Reply:
Fixed!